src/statmodels.jl

```diff
@@ -160,17 +160,50 @@ df_residual(obj::RegressionModel) = error("df_residual is not defined for $(type
 ## Nms are the coefficient names, corresponding to rows in the table
 type CoefTable
     mat::Matrix
+    cols::Tuple
     colnms::Vector
     rownms::Vector
-    pvalcol::Integer
-    function CoefTable(mat::Matrix,colnms::Vector,rownms::Vector,pvalcol::Int=0)
-        nr,nc = size(mat)
-        0 <= pvalcol <= nc || error("pvalcol = $pvalcol should be in 0,...,$nc]")
+    function CoefTable(cols::Tuple,colnms::Vector,rownms::Vector)
+        nc = length(cols)
+        nrs = map(length,cols)
```

Comment:
I would simply assert that all columns are of equal length, and then use `nr` as before. I can't see a use case for providing columns with different lengths (which is more likely to be a caller error).